Bugzilla/User.pm

```diff
@@ -176,21 +176,28 @@ sub ES_PROPERTIES {
 sub es_document {
     my ( $self, $timestamp ) = @_;
     my $doc = {
-        login          => $self->login,
-        name           => $self->name,
-        is_enabled     => $self->is_enabled,
+        login        => $self->login,
+        name         => $self->name,
+        is_enabled   => $self->is_enabled,
         suggest_user => {
             input => [ $self->login, $self->name ],
             output => $self->identity,
             payload => { name => $self->login, real_name => $self->name },
         },
     };
-    if ($self->name && $self->name =~ /:(\w+)/) {
-        my $ircnick = $1;
+    my @nicks;
+    if ($self->name) {
```

Comment:
`my(@nicks) = ($self->name =~ m//mg);` being the alternate form I envisioned for 188...